app/code/Magento/Checkout/ViewModel/Cart.php

```diff
@@ -0,0 +1,49 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\Checkout\ViewModel;
+
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\View\Element\Block\ArgumentInterface;
+use Magento\Framework\View\Element\Context;
+use Magento\Store\Model\ScopeInterface;
+
+class Cart implements ArgumentInterface
+{
+    /**
+     * Config settings path to enable clear shopping cart button
+     */
+    private const XPATH_CONFIG_ENABLE_CLEAR_SHOPPING_CART = 'checkout/cart/enable_clear_shopping_cart';
+
+    /**
+     * @var ScopeConfigInterface
+     */
+    private $_scopeConfig;
+
+    /**
+     * Constructor
+     *
+     * @param Context $context
+     */
+    public function __construct(
+        Context $context
+    ) {
+        $this->_scopeConfig = $context->getScopeConfig();
+    }
+
+    /**
+     * Check if clear shopping cart button is enabled
+     *
+     * @return bool
+     */
+    public function isClearShoppingCartEnabled()
+    {
+        return (bool) $this->_scopeConfig->getValue(
+            self::XPATH_CONFIG_ENABLE_CLEAR_SHOPPING_CART,
+            ScopeInterface::SCOPE_STORE
```

Comment:
This should be SCOPE_WEBSITE based on the issue AC
> Merchant can configure option by website scope (example: enabled for B2B site, disabled for B2C site)

Reply:
Updated to SCOPE_WEBSITE